pkg/daemon/ceph/client/pool_test.go

```diff
@@ -26,6 +26,10 @@ import (
 	"github.com/rook/rook/pkg/clusterd"
 )
 
+const (
```

Comment:
Although this does not affect your test, it's usually a good practice not to use constant in test files so that they can be changed in the tests.

Reply:
@leseb : Thanks for the inputs. I've removed the constant from the test file.